rust/src/combinators/select_private_candidate/references.bib

```diff
@@ -0,0 +1,9 @@
+@misc{liu2018privateselectionprivatecandidates,
+      title={Private Selection from Private Candidates}, 
+      author={Jingcheng Liu and Kunal Talwar},
+      year={2018},
+      eprint={1811.07971},
+      archivePrefix={arXiv},
+      primaryClass={cs.DS},
+      url={https://arxiv.org/abs/1811.07971}, 
```

Comment:
Surprised that this doesn't render as a link in the References section of the PDF?

Reply:
Agree, it's annoying. I'll make a pass through and update bib styling for everything at some point.